docs/integrations/sources/alloydb.md

```diff
@@ -0,0 +1,13 @@
+# AlloyDB for PostgreSQL
```

Comment:
please update this readme file according to the new connectors template

Reply:
done